OpenSubtitlesHandler/MethodResponses/MethodResponseAutoUpdate.cs

```diff
@@ -17,7 +17,6 @@
    along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-using System.ComponentModel;
```

Comment:
newline here, but after this it's 2 newlines. Maybe add one up there and remove one here :)